lstmcpipe/config/pipeline_config.py

```diff
@@ -161,12 +162,12 @@ def complete_lstmcpipe_config(loaded_config):
 
         # TODO parse version from hiPeRTA module
         import lstchain
-        base_prod_id = f"{year}{month}{day}_vRTA420_v{lstchain.__version__}"
+        base_prod_id = f"{year}{month}{day}_vRTA420_v{lstchain.__version__}:"
```

Comment:
Why two points and only in the hiperta case ?

Reply:
typo